napari/layers/points/points.py

```diff
@@ -305,12 +305,9 @@ def __init__(
 
         self._drag_start = None
 
-        # Nx2 array of points in the currently viewed slice
-        self._data_view = np.empty((0, 2))
-        # Sizes of points in the currently viewed slice
-        self._size_view = 0
-        # Full data indices of points located in the currently viewed slice
+        # initialize view data
         self._indices_view = []
+        self._point_scale_view = []
```

Comment:
can this be `_view_size_scale` as it is really a scalar on the size of the points right?

Reply:
That's a good point. Changed!